mono/metadata/verify.h

```diff
@@ -8,7 +8,6 @@
 #include <mono/metadata/metadata.h>
 #include <mono/metadata/image.h>
 #include <mono/metadata/loader.h>
-#include <mono/metadata/class-internals.h>
 #include <glib.h> /* GSList dep */
```

Comment:
If this is a public header, it shouldn't be depending on glib.h either, should it?